server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java

```diff
@@ -185,7 +185,6 @@ public Object getProperty(String key) {
   public static class StringFieldBuilder {
     private final NewIndexType indexType;
     private final String fieldName;
-    private boolean docValues = false;
     private boolean disableSearch = false;
     private boolean hasAnalyzedField = false;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "hasAnalyzedField" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1068)